setup.py

```diff
@@ -3,9 +3,9 @@
     name='consensus',
     version='1.0',
     packages=find_packages(),
-    install_requires=['molgenis-py-client>=2.1.0', 'progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
+    install_requires=['progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
     author='Mariska Slofstra',
     license='GNU Lesser General Public License 3.0',
```

Comment:
I think we generally use the "Lesser" version nowadays, but I suggest checking this before actually changing.